File: Java/CWE-798.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class HardCodedCredentialsExample {

    public static void main(String[] args) {
        String dbUrl = "jdbc:mysql://localhost:3306/mydatabase";
        String username = "admin"; // Hard-coded username
        String password = "password123"; // Hard-coded password

        try {
            Connection connection = DriverManager.getConnection(dbUrl, username, password);
            System.out.println("Connected to the database successfully!");
        } catch (SQLException e) {
            System.out.println("Failed to connect to the database.");
            e.printStackTrace();
        }
    }
}

File: C#/CWE-798.cs
using System;
using System.Data.SqlClient;

class Program
{
    static void Main()
    {
        string connectionString = "Server=myServerAddress;Database=myDataBase;User Id=admin;Password=password123;"; // Жёстко закодированные учетные данные

        using (SqlConnection connection = new SqlConnection(connectionString))
        {
            try
            {
                connection.Open();
                Console.WriteLine("Connection successful!");
            }
            catch (Exception ex)
            {
                Console.WriteLine("Connection failed: " + ex.Message);
            }
        }
    }
}
